<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>791</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>791</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>天气查询小工具</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>791</width>
      <height>81</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>791</width>
      <height>81</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/head_pic.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>所查城市：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="weatherComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">;</string>
    </property>
    <item>
     <property name="text">
      <string>北京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上海</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>天津</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重庆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>哈尔滨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>河北</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>河南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>安徽</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>浙江</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>福建</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>甘肃</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>广东</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>广西</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>贵州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>云南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>内蒙古</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>江西</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>湖北</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四川</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>宁夏</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>青海</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>山东</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>山西</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>陕西</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新疆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西藏</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>台湾</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>海南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>湖南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>江苏</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黑龙江</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>吉林</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>辽宁</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="resultText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>791</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>791</width>
      <height>281</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>791</width>
      <height>281</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background/images/concent_pic.png);
color: rgb(0, 255, 0);</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="queryButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ewm">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/weather_web.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_ewmjs">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>中国天气网</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>791</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>791</width>
      <height>121</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>791</width>
      <height>121</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background/images/tail_pic.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>更换壁纸：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PiccomboBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>concent_pic.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>concent_pic2.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>concent_pic3.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>concent_pic4.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>concent_pic5.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>concent_pic6.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>concent_pic7.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>concent_pic8.png</string>
     </property>
    </item>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>weatherComboBox</zorder>
   <zorder>resultText</zorder>
   <zorder>queryButton</zorder>
   <zorder>clearButton</zorder>
   <zorder>label_ewm</zorder>
   <zorder>label_ewmjs</zorder>
   <zorder>label_6</zorder>
   <zorder>PiccomboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>images/退出logo.png</normaloff>images/退出logo.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>小游戏</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>queryWeather()</slot>
  <slot>clearResult()</slot>
 </slots>
</ui>
